styles.css

```diff
@@ -58,4 +65,137 @@
     gap: 16px;
     list-style-type: none;
   }
+
+  .works {
+    display: grid;
+    grid-template-rows: repeat(4, 1fr);
+    grid-template-columns: repeat(1fr);
+    margin-top: 50px;
+    gap: 50px;
+  }
+
+  .Tonic {
+    color: #172b4d;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .canopi {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #344563;
+  }
+
+  .backend {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+  }
+
+  .fiftin {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+  }
+
+  .art {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .frontend {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    color: #6070ff;
+    list-style-type: none;
+    display: flex;
+    gap: 16px;
+  }
+
+  .frontend li {
+    background: #ebebff;
+    padding: 4px 12px;
+    border-radius: 8px;
+  }
+
+  .seeproject {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #396df2;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    background-color: inherit;
+  }
+
+  .card1 {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 16px;
+    border-radius: 16px;
+    border: solid #e5e5e5 1px;
+    background-color: #fff;
+  }
+
+  .about {
```

Comment:
Kindly update the styling of your about myself section to better match the figma design🙏.
Your design:
<img width="235" alt="image" src="https://user-images.githubusercontent.com/37116322/219485377-dfdbf68b-7a6b-421c-8072-3943232a4cca.png">

Figma design:
<img width="190" alt="image" src="https://user-images.githubusercontent.com/37116322/219485428-d820d22b-1772-4fd4-877c-38a6bd67ad5b.png">

A few things to note:
- Your design is currently lacking padding at the top of the section and on the left and right sides
- The social links, should be aligned to the left
- The Get my resume button is missing padding on all sides
- Also, please add some space between the technology icons and the technology names in the languages section to better match the figma design as seen in the screenshot below:
<img width="166" alt="image" src="https://user-images.githubusercontent.com/37116322/219486552-34919bc3-5d8b-41c9-bc22-e2807a7c3a0d.png">


**Please try your best to make sure that your design matches the Figma design as much as possible😎**